pallets/aleph/src/lib.rs

```diff
@@ -381,6 +443,51 @@ pub mod pallet {
 
             Ok(())
         }
+
+        // fix weight
+        #[pallet::call_index(3)]
+        #[pallet::weight(T::BlockWeights::get().max_block * Perbill::from_percent(10))]
+        /// Stores abft score
+        pub fn unsigned_submit_abft_score(
+            origin: OriginFor<T>,
+            score: Score,
+            _signature: SignatureSet<Signature<T>>, // We don't check signature as it was already checked
```

Comment:
This looks very suspicious, even while I think it's correct. Could you a bit more explicitly say that it was checked in `pre_dispatch` and that it's impossible to bypass that at any point in block production or import (as long as that is correct ofc)?

Reply:
Done.